develop/tutorials/articles/03-portlets/packaging-a-jsf-application.markdown

```diff
@@ -20,16 +20,12 @@ JSF framework expects a WAR layout and often requires the
 a JSF portlet so it can be treated as an OSGi module at runtime, and the pros
 and cons behind both.
 
-Liferay 7.0 supports the OSGi WAB standard for deployment of Java EE style WARs.
-Simply put, a WAB is an archive that that has a WAR layout and contains a
+Liferay 7.0 supports the OSGi Web Archive Bundle (WAB) standard for deployment of Java EE style WARs.  Simply put, a WAB is an archive that has a WAR layout and contains a
```

Comment:
See Section 128.3 of the OSGi v6 Specification, titled "Web Application Bundle" https://osgi.org/download/r6/osgi.enterprise-6.0.0.pdf